src/main/webapp/app/guided-tour/tours/tutor-dashboard-tour.ts

```diff
@@ -0,0 +1,101 @@
+import { GuidedTour } from 'app/guided-tour/guided-tour.model';
+import { ImageTourStep, TextTourStep } from 'app/guided-tour/guided-tour-step.model';
+import { Orientation, UserInteractionEvent } from 'app/guided-tour/guided-tour.constants';
+
+export const tutorCourseDashboardTour: GuidedTour = {
+    courseShortName: 'artemistutorial',
+    exerciseShortName: 'Patterns in Software Engineering',
+    settingsKey: 'tutor_course_dashboard_tour',
+    steps: [
+        new TextTourStep({
+            highlightSelector: '.tutor-stats',
```

Comment:
In my opinion it is better to use id selectors instead of dummy classes. What do you think @lovaiible ?